### PR TITLE
DX-323 Fix issue to load accounts on Accounts Verification starterKit

### PR DESCRIPTION
**Pull request by basiq-ash**
- https://basiq.atlassian.net/browse/DX-323

### CODE REVIEW

File: components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js
Comment (jayden-basiq) [inline components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js:1]: should we update to use the `jobIds` param here instead?

Comment (basiq-ash) [reply components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js:1]: @jayden-basiq The issue was about the filters that was invalidating while fetching accounts. I am fetching jobId correctly and this app will never need jobIds.

Comment (jayden-basiq) [reply components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js:1]: can be in future update all good though!

**Review by jayden-basiq — APPROVED** [review]
LGTM

**Pull request merged**